public/app/index.html

```
@@ -320,6 +320,9 @@ <h1 class="hdr-title">
           <button class="btn btn-preset" data-preset="restoration">
             Restoration
           </button>
+          <button class="btn btn-preset" data-preset="clearvoice" title="Removes ALL non-voice sound. Max ML separation + hard spectral gate. -96 dB noise floor.">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)